src/pages/CreatePolicyWizard/__tests__/CreatePolicyWizard.test.tsx

```diff
@@ -38,32 +41,32 @@ describe('src/pages/ListPage/CreatePolicyWizard', () => {
 
         // eslint-disable-next-line testing-library/no-node-access
         return <ClientContextProvider client={ client }>{ props.children }</ClientContextProvider>;
-    };
+    }});
 
-    const failIfNoHttpCallMatched = () => {
-        const calls = fetchMock.calls(UNMATCHED).filter(c => c.isUnmatched);
-        if (calls.length > 0) {
-            throw new Error(`Found ${ calls.length } unmatched calls, maybe you forgot to mock? : ${calls.map(c => c.request?.url || c['0'])}`);
-        }
-    };
+    // const failIfNoHttpCallMatched = () => {
+    //     const calls = fetchMock.calls(UNMATCHED).filter(c => c.isUnmatched);
+    //     if (calls.length > 0) {
+    //         throw new Error(`Found ${ calls.length } unmatched calls, maybe you forgot to mock? : ${calls.map(c => c.request?.url || c['0'])}`);
+    //     }
+    // };
```

Comment:
```suggestion
```

We can remove this commented block. There is a similar check on the `AppWrapper`